SDKBuildScripts/Update-PlayFabSdk.ps1

```diff
@@ -0,0 +1,160 @@
+[CmdletBinding(SupportsShouldProcess = $true, DefaultParameterSetName = "ApiSpecGitUrl")]
+param(
+    [Parameter(Position = 1)]
+    [ValidateSet(
+        "ActionScriptSDK",
+        "Cocos2d-xSDK",
+        "CSharpSDK",
+        "JavaSDK",
+        "JavaScriptSDK",
+        "LuaSDK",
+        "NodeSDK",
+        "Objective_C_SDK",
+        "PhpSDK",
+        "PostmanCollection",
+        "PythonSDK",
+        "SdkTestingCloudScript",
+        "UnrealMarketplacePlugin",
+        "UnitySDK",
+        "WindowsSDK",
+        "XPlatCppSDK")]
+    [string[]]$SdkNames,
+    [Parameter(ParameterSetName="ApiSpecPath")]
+    [string]$ApiSpecPath,
+    [Parameter(ParameterSetName="ApiSpecPfUrl")]
+    [string]$ApiSpecPfUrl,
+    [Parameter(ParameterSetName="ApiSpecPfUrlBranch")]
+    [string]$ApiSpecBranch,
+    [Parameter(ParameterSetName="ApiSpecGitUrl")]
+    [switch]$ApiSpecGitUrl,
+    [switch]$KeepSource,
+    [switch]$NonNullable,
+    [switch]$Beta
+)
+
+begin
+{
+    if(!(Get-Command node))
+    {
+        throw "You must have Node.js installed to generate the SDK"
+    }
+
+    $sdkTargetSrcMap = @{
```

Comment:
I might come back to you later and ask that you read this information from a different place. But it's certainly useful now.

Reply:
Sounds good to me!